src/benchmark.rs

```diff
@@ -23,6 +23,7 @@ fn thread_func(benchmark: Arc<Vec<Box<(Runnable + Sync + Send)>>>, iterations: i
     context.insert("iteration".to_string(), Yaml::String(iteration.to_string()));
     context.insert("thread".to_string(), Yaml::String(thread.to_string()));
     context.insert("base".to_string(), Yaml::String(base.to_string()));
+    context.insert("no_check_certificate".to_string(), Yaml::Boolean(no_check_certificate));
```

Comment:
I'm not 100% sure if this should be a context option.
Context variables are available to be interpolated in urls and headers and `no_check_certificate` one is not giving so much value for them, but I understand that send this option to executable items could be a pain.

Reply:
Yeah, maybe I should change it to pass a `config` object around that has all the global configurations (which, for now, is mostly this one option, I suppose).

Reply:
I like the idea. It's good to share this object with these kind of program flags. For example the base_path and the reports_path could go there too.